components/financial_assistance/spec/models/financial_assistance/applicant_spec.rb

```diff
@@ -240,6 +240,39 @@
     end
   end
 
+  describe '#tribe_codes' do
+    context "when tribe codes is set to nil" do
+      it "should return an empty array" do
+        applicant.update_attributes(tribe_codes: nil)
```

Comment:
FYI, I've found it more efficient to use the `.set` method in these kinds of specs instead of `update_attributes`.  I'm assuming there's no need for callbacks/validations to be fired here, so using `.set` will avoid that and make the test run faster.